map/generate-offline-map-overrides/README.md

```diff
@@ -1,35 +1,26 @@
-# Generate Offline Map (Overrides)
+# Generate offline map (overrides)
 
 Take a web map offline with additional options for each layer.
 
-For applications where you just need to take all layers offline, use the standard workflow (using only `GenerateOfflineMapParameters`). For a simple example of how you take a map offline, see the "Generate Offline Map" sample. For more fine-grained control over the data you are taking offline, use overrides to adjust the settings for each layer. Some example use cases for the overrides approach could be when you need to:
+![Image of generate offline map overrides](GenerateOfflineMapOverrides.png)
 
-* adjust the extent for one or more layers to be different to the rest of the map.
-* reduce the amount of data (such as tiles) downloaded for one or more layers in the map.
-* filter features to be taken offline.
-* take features with no geometry offline.
+## Use case
 
-![](GenerateOfflineMapOverrides.png)
+When taking a web map offline, you may adjust the data (such as layers or tiles) that is downloaded by using custom parameter overrides. This can be used to reduce the extent of the map or the download size of the offline map. It can also be used to highlight specific data by removing irrelevant data. Additionally, this workflow allows you to take features offline that don't have a geometry - for example, features whose attributes have been populated in the office, but still need a site survey for their geometry.
```

Comment:
```suggestion
When taking a web map offline, you may adjust the data (such as layers or tiles) that are downloaded by using custom parameter overrides. This can be used to reduce the extent of the map or the download size of the offline map, or to highlight specific data by removing irrelevant data. Additionally, this workflow allows you to take features offline that don't have a geometry; for example, features whose attributes have been populated in the office, but still need a site survey for their geometry.
```